src/avram/migrator/alter_table_statement.cr

```diff
@@ -36,14 +54,19 @@ class Avram::Migrator::AlterTableStatement
   end
 
   def statements
-    [alter_statement] + index_statements + fill_existing_with_statements
+    alter_statements + change_type_statements + index_statements + fill_existing_with_statements
   end
 
-  def alter_statement
-    String.build do |statement|
-      statement << "ALTER TABLE #{@table_name}"
-      statement << "\n"
-      statement << (rows + dropped_rows).join(",\n")
+  def alter_statements : Array(String)
+    if (rows + dropped_rows).empty?
```

Comment:
If there  are no alter statements then we should generate an empty `ALTER` or it will error